sdk/src/main/kotlin/io/rover/sdk/Rover.kt

```diff
@@ -83,85 +83,77 @@ import java.util.concurrent.Executor
 
 /**
  * Entry point for the Rover SDK.
- *
- * The Rover SDK consists of several discrete modules, which each offer a major vertical
- * (eg. Experiences and Location) of the Rover Platform.  It's up to you to select which
- * are appropriate to activate in your app.
- *
- * Serves as a dependency injection container (a sort of backplane) for the various components of
- * the Rover SDK.
  */
+
 open class Rover(
     /**
      * When initializing Rover you must give it a reference
      */
-    open val application: Application,
+    private val application: Application,
 
     /**
      * Set your Rover Account Token (API Key) here.
      */
-    open var accountToken: String? = null,
+    private var accountToken: String? = null,
```

Comment:
hm not sure why I left it like that but try seeing if you can make this not nullable and remove default value.